src/runtime.rs

```diff
@@ -246,6 +249,12 @@ impl Runtime {
         },
         None => Box::new(disk_fs::DiskFsStore::new()),
       },
+      acme_store: match settings.acme_store {
+        Some(ref store) => match store {
+          AcmeStore::Redis(conf) => Some(Box::new(redis_acme::RedisAcmeStore::new(&conf))),
+        },
+        None => None,
+      },
```

Comment:
Should we just make this a global for now? It's fine as-is, but it's going to use some heap and a pool for every runtime, to do something that isn't runtime-specific.